pkg/agent/testdata/RawUbuntu/line9.sh

```diff
@@ -228,7 +228,7 @@ version_gte() {
 }
 
 systemctlEnableAndStart() {
-    systemctl_restart 100 5 30 $1
+    systemctl_restart 100 5 300 $1
```

Comment:
I'm a bit concerned about this stretching out the time before service start failures are detected for everything else invoking systemctlEnableAndStart.

Could this timeout be parameterized so that every other invocation maintains the 30s timeout?

Reply:
Have reverted this change as it is no longer needed for the latest R525 driver version.